openbox/config.c

```diff
@@ -50,6 +50,7 @@ gboolean config_theme_keepborder;
 guint    config_theme_window_list_icon_size;
 
 gchar   *config_title_layout;
+gboolean config_apply_default_icon;
```

Comment:
could we have a consistent ordering across files? this can go right below `config_theme_window_list_icon_size` here, so move up a couple lines.